### ISSUE TITLE
Differences in actual minimum module ages within same year

### ISSUE THREAD

**Issue by buscandoaverroes** [worldbank/gld#50]
In theory, the minimum applicable ages for the labor and the education modules should be same within the same year (unless a change is documented). But there are a few years in the data that show one particular round with much lower age data than the rest of the rounds in the same year. See year 2017 with `pufc23_pclass` as an example. I want to talk more about this during meetings but for documentation purposes I also want to record the issue here.

What should happen in cases where rounds in the same year display different age minimums? My instinct is to simply document this and code the year's minimum age for the module as the **lowest age recorded in any wave that year-module**. Doing this would preserve all observations.

**Comment by gronert-m** [comment]
@buscandoaverroes I suggest to use the 15 years as the minimum. From what I saw 15 years is the age limit for `c23_class` for the first three quarters, then switches to 5 for some odd reason. We'd be basing then the info for those 5 to 14 on one quarter alone (in itself possible). In addition, I have looked at the bulletins for July 2016 and July 2019 (the closest available) and in both cases information is reported for the population 15 or above - not just labour information but overall age distribution of the population and education levels, so I think we can be confident in saying that 15 was the age they were aiming to publish at.

On the bulletin note I suggest you getting in touch with the PSA as well, in the hopes of having more luck with obtaining info. I have outsourced that to issue #51 so it does not interfere with this which, if you are OK with my proposal (happy to discuss more) we may close.
